Description: Fix spelling error in manpages
 This patch fix three spelling errors in documentation and comments.
Author: Gennaro Oliva <oliva.g@na.icar.cnr.it>
Bug: https://bugs.schedmd.com/show_bug.cgi?id=8842
Forwarded: https://bugs.schedmd.com/show_bug.cgi?id=8842
Last-Update: 2020-04-13

--- slurm-llnl-20.02.1.orig/contribs/perlapi/libslurm/perl/lib/Slurm.pm
+++ slurm-llnl-20.02.1/contribs/perlapi/libslurm/perl/lib/Slurm.pm
@@ -332,7 +332,7 @@ Get the string representation of the spe
 
 =head3 $resp = $slurm->allocate_resources($job_desc);
 
-Allocate resources for a job request. If the requested resources are not immediately available, the slurmctld will send the job_alloc_resp_msg to the sepecified node and port.
+Allocate resources for a job request. If the requested resources are not immediately available, the slurmctld will send the job_alloc_resp_msg to the specified node and port.
 
 =over 2
 
--- slurm-llnl-20.02.1.orig/doc/man/man5/slurm.conf.5
+++ slurm-llnl-20.02.1/doc/man/man5/slurm.conf.5
@@ -1190,7 +1190,7 @@ NOTE: More information is available at t
 ( https://slurm.schedmd.com/elasticsearch.html ).
 The value "jobcomp/filetxt" indicates that a record of the job should be
 written to a text file specified by the \fBJobCompLoc\fR parameter.
-The value "jobcomp/lua" indicates that a record of the job should procesed by
+The value "jobcomp/lua" indicates that a record of the job should processed by
 the "jobcomp.lua" script located in the default script directory
 (typically the subdirectory "etc" of the installation directory).
 The value "jobcomp/mysql" indicates that a record of the job should be
--- slurm-llnl-20.02.1.orig/slurm/slurm.h.in
+++ slurm-llnl-20.02.1/slurm/slurm.h.in
@@ -3175,7 +3175,7 @@ extern void slurm_init_job_desc_msg(job_
 /*
  * slurm_allocate_resources - allocate resources for a job request
  *   If the requested resources are not immediately available, the slurmctld
- *   will send the job_alloc_resp_msg to the sepecified node and port.
+ *   will send the job_alloc_resp_msg to the specified node and port.
  * IN job_desc_msg - description of resource allocation request
  * OUT job_alloc_resp_msg - response to request.  This only represents
  *   a job allocation if resources are immediately.  Otherwise it just contains
